test/data/adult-test.txt

```diff
@@ -0,0 +1,119 @@
+#@ TextLoader{
```

Comment:
Quite strange...

As @eerhardt mentioned, this was removed in #1687. But in out tests we are still using it. We're pulling from the history of our git repo, but it's no longer in master.

https://github.com/dotnet/machinelearning/blob/053062baf461b39622465de6b06326df1d1d3435/src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs#L26-L30

Reply:
We aren't using it in our tests - just in our samples.

https://github.com/dotnet/machinelearning/search?q=DownloadAdultDataset&unscoped_q=DownloadAdultDataset

Reply:
As we're calling from our samples, should we bring files back into the main branch (instead of pulling from the repo's history)?